tracer/src/Datadog.Trace/Debugger/LiveDebuggerFactory.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="LiveDebuggerFactory.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Agent.DiscoveryService;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Debugger.Configurations;
+using Datadog.Trace.Debugger.ProbeStatuses;
+using Datadog.Trace.Debugger.Sink;
+using Datadog.Trace.RemoteConfigurationManagement;
+
+namespace Datadog.Trace.Debugger;
+
+internal class LiveDebuggerFactory
+{
+    public static LiveDebugger Create(IDiscoveryService discoveryService, IRemoteConfigurationManager remoteConfigurationManager, string serviceName)
```

Comment:
Why do we need a separate class for this, as opposed to just putting it inside `LiveDebugger`? Seems that it creates a situation where our initialization logic is spread across this method and `LiveDebugger.Create`, which doesn't seem necessary?

Reply:
As discussed, I'm aligning with other name conventions in code, for example `TracerManagerFactory`

Reply:
That's fair, consistency should come first.